src/sourmash/minhash.py

```diff
@@ -295,6 +295,23 @@ def add_sequence(self, sequence, force=False):
         self._methodcall(lib.kmerminhash_add_sequence, to_bytes(sequence),
                          force)
 
+    def seq_to_hashes(self, sequence, force=False, is_protein = False):
+        "Convert sequence to hashes without adding to the sketch."
+
+        if is_protein and self.moltype not in ["protein", "dayhoff"]:
+            raise ValueError("Expected DNA sequence, found Protein sequence.")
+
+        size = ffi.new("uintptr_t *")
+        hashes_ptr = self._methodcall(lib.kmerminhash_seq_to_hashes, to_bytes(sequence), len(sequence), force, is_protein, size)
+        size = size[0]
+
+        try:
+            d = ffi.unpack(hashes_ptr, size)
+            return _HashesWrapper([k for k in d])
```

Comment:
I _think_ we should probably leave it as a list, and not wrap it in `_HashesWrapper`.

One rationale for this is that if we leave it as a list, the positions of the hashes in the list will match the positions of the k-mers in the original sequence, which seems like it could be useful.

Reply:
(and, if one _wanted_ the `_HashesWrapper`, it's easy enough to ask for it.)

Reply:
I tried it and the order is still the same because the wrapper iterate and insert the hashes in the same order. I will remove the wrapper as it's not required though.

Reply:
ok - is it a dictionary when `track_abundance=True`, though?

Reply:
I did not design it to track abundance. Do we need to `track_abundance` in `seq_to_hashes`?

Reply:
I agree it should not track abundance. I think I got confused about what was going on, so never mind :)